packages/contracts/test/multi_sig_with_time_lock_except_remove_auth_addr.ts

```diff
@@ -165,17 +168,17 @@ describe('MultiSigWalletWithTimeLockExceptRemoveAuthorizedAddress', () => {
             const txHash = await multiSigWrapper.submitTransactionAsync(validDestination, owners[0], dataParams);
```

Comment:
Looks like `multiSigWrapper` is only used for calling  `submitTransactionAsync`. Is this not included in the auto-generated wrapper classes?

Reply:
This function needs to code the params before passing them down